src/main/webapp/frame.jsp

```diff
@@ -12,71 +12,18 @@
   <!-- CSS links -->
   <!-- Latest compiled and minified CSS -->
   <!-- <link href="css/font-awesome.min.css" rel="stylesheet" type="text/css"/> -->
-  <link ng-href="css/loading.css" rel="stylesheet" type="text/css"/>
   <link ng-href="css/themes/uw-{{portal.theme.name}}.${project.version}.css" rel="stylesheet" type="text/css"/>
   <link href="my-app/my-app.css" rel="stylesheet" type="text/css"/>
   <link rel="shortcut icon" href="bower_components/uw-ui-toolkit/dist/img/favicon.ico" type="image/x-icon"/>
 </head>
 
-<body ng-controller="PortalMainController as mainCtrl" class='my-uw'>
-  <div ng-hide="portal.loading.hidden" ng-class="{slowfade: portal.loading.startFade}" style=" 
-          position : fixed;
-          top: 0;
-          z-index: 2000;
-          color: darkgrey;
-          background: #EAE8DF;
-          height: 100vh;
-          width: 100vw;
-          text-align: center;
-          padding-top: 45vh;
-          font-size: xx-large;
-          margin: -8px;
-          font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;">
-          Loading...</div>
-  <div class='sr-only' ng-if="classicURL">
-    <a ng-href='{{classicURL}}'>Switch back to the classic MyUW</a>
-  </div>
-  <div ng-controller="PortalPopupController as popupCtrl">
-    <!--[if lt IE 10]>
-    <div class="browserupgrade">
-      <span class="fa fa-frown-o"></span>
-      <p>Sorry, MyUW beta does not support your browser.<br/><a href="https://kb.wisc.edu/myuw/page.php?id=51345">Learn how to upgrade your browser.</a><br/>Can't upgrade? <a href="http://my.wisc.edu/portal/Login?profile=default">Switch back to MyUW classic.</a></p>
-    </div>
-    <![endif]-->
-    <noscript>
-        <div class="alert alert-warning alert-dismissible" role="alert" style="margin-bottom:0;">
-        <div class="container">
-          <i class="fa fa-2x fw fa-exclamation-triangle pull-left"></i>
-          <strong>
-            Please <a href="http://enable-javascript.com" target="_blank">enable Javascript</a> to interact with all forms and features on our website.  For further assistance, contact the <a href="https://kb.wisc.edu/helpdesk/" target="_blank">DoIT Help Desk</a>.
-          </strong>
-        </div>
-      </div>
-    </noscript>
+<body>
```

Comment:
This is awesome, makes that JSP a lot easier to reproduce for other languages.

:+1: x 10.